src/Migration/Migration1686772873AddActiveConfig.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace Frosh\ThumbnailProcessor\Migration;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+class Migration1686772873AddActiveConfig extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1686772873;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $configKey = 'FroshPlatformThumbnailProcessor.config.';
```

Comment:
You can use also systemConfigService with ->getBoolean('FroshPlatformThumbnailProcessor.config. Active', true);

as all new installations write that automaticalls to false, it should give false and for non existient keys true

Reply:
But while the defaultValue is false, false will be saved when plugin is updated. So this would not work :-(